src/instance.js

```diff
@@ -244,7 +244,7 @@ fakeModelInstance.prototype.save = function () {
 	return this.validate().then(function (err) {
 		if(err)
 			throw err;
-		
+		self.options.isNewRecord = false;
```

Comment:
Probably also want to add `self.isNewRecord` as well here. That seems to be the one actually documented, but both are supported